### DISCUSSION TITLE
r19base:/9.loadmore

### DISCUSSION THREAD

**Discussion by giscus** [yangbo5207/react19feature#14] [General]
# tutorial/use/loadmore

http://localhost:5173/tutorial/use/loadmore

**Comment by yangbo5207** [comment]
欢迎各位道友添加我的微信好友 icanmeetu 交流学习心得

**Comment by dlgchg** [comment]
我来了

**Comment by 0bipinnata0** [comment]
第一次见这种[List,List,List]用作loading more的，学习了。
在RN上面可以用sectionList来实现类似的功能

**Comment by rmstsd** [comment]
感觉在列表里显示数据的 index 会有些麻烦
